packages/components/src/utils/clear-selection.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Clears the text selection on screen.
+ */
+export function clearSelection() {
+	if ( window?.getSelection ) {
+		// eslint-disable-next-line @wordpress/no-global-get-selection
+		const selection = window?.getSelection();
+		if ( ! selection ) {
+			return;
+		}
+
+		if ( selection.empty ) {
+			// Chrome.
+			selection.empty();
+		} else if ( selection.removeAllRanges ) {
+			// Firefox.
+			selection.removeAllRanges();
+		}
+		// @ts-ignore IE specific property.
+	} else if ( document.selection ) {
+		// @ts-ignore IE specific property.
+		document.selection.empty();
+	}
+}
```

Comment:
Should this actually go in the `dom` package?

Reply:
I wasn't sure about a lot of these utils that seems to be only required by TextInput. 

I was hoping you could advise! 😆 

If you believe that it would be useful to expose this method, and `dom` is the correct place then I'll take your advice :)